### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by motivforz** [petretiandrea/home-assistant-tapo-p100#239] [General]
Just wanted to write a note to say thank you for the work on this integration. I've been using it for a few months after my previous P100 integration had an issue, and it's been excellent. Just updated to the latest version and the update was seamless, when previously it's fallen over and I've had to re-add all devices. Again - thank you very much!

**Comment by apotkamp** [comment]
And also thank you from me! 
Works fine on my P110 device on a home assistant core installation on a raspberry pi 3b.
